### PR TITLE
Added external links to the search for movies.

### LINKED ISSUE #4583: Adding more info to Discover screens
**Is your feature request related to a problem? Please describe.**
Adding more information to the screens within Discover to make it more useful. In the screen below it would be useful to be able to add it to the exclusion list. At present you have to close it, select the movie on the list and then click the exclusion button at the bottom

image

It would also be great if this screen included a link to IMDB/TMDB and also included the Director/Main Actors. At present I look at the list and then have to search for them manually in IMDB to see whether they're of interest

### PR DESCRIPTION
**Pull request by austinwbest**
#### Database Migration
NO

#### Description
Adds external links to search results table. Original request for putting on the modal i dont think is realistic. To much information on the modal (also wanted actors/director) so adding the external links to the result table saves clicking each one to get to the links and having to cancel to get back to the table. 

#### Issues Fixed or Closed by this PR

Fixes #4583

### CODE REVIEW

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js [hunk 1]
```diff
@@ -117,17 +118,41 @@ class AddNewMovieSearchResult extends Component {
...
-                onPress={this.onTMDBLinkPress}
-              >
-                <Icon
-                  className={styles.tmdbLinkIcon}
-                  name={icons.EXTERNAL_LINK}
-                  size={28}
-                />
-              </Link>
+              {
+                isSmallScreen ?
```
Comment (Qstick) [inline frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js:122]: Shouldn't be needed now.

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js [hunk 2]
```diff
@@ -156,6 +181,42 @@ class AddNewMovieSearchResult extends Component {
               }
             </div>
 
+            {
+              isSmallScreen ?
```
Comment (Qstick) [inline frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js:185]: Shouldn't be needed now

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js [hunk 3]
```diff
@@ -156,6 +181,42 @@ class AddNewMovieSearchResult extends Component {
...
+                <div className={styles.externalLink}>
+                  <Link
+                    to={`https://www.themoviedb.org/movie/${tmdbId}`}
+                    onPress={this.onExternalLinkPress}
+                  >
+                    <Label size={sizes.LARGE}>
+                      TMDB
```
Comment (Qstick) [inline frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js:192]: TMDb

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js [hunk 4]
```diff
@@ -156,6 +181,42 @@ class AddNewMovieSearchResult extends Component {
...
+
+                  {
+                    imdbId &&
+                      <Link
+                        to={`https://www.imdb.com/title/${imdbId}`}
+                        onPress={this.onExternalLinkPress}
+                      >
+                        <Label size={sizes.LARGE}>
+                          IMDB
+                        </Label>
```
Comment (Qstick) [inline frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js:204]: IMDb

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js (hunk 1 above)
Comment (austinwbest) [reply frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js:122]: Needed to move the row of buttons down for mobile devices

File: frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js (hunk 3 above)
Comment (austinwbest) [reply frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js]: Fixed

**Pull request merged**